backpack.tf extended sorting.user.js

```diff
@@ -1268,4 +1268,246 @@ class</a></li>
 		childList: true
 	});
 
+    // ################# Killstreak Kit Additions for Stats pages ##############
+
+    function trimSpaces(string){
+        return string.replace(/^\s+/g, '').replace(/\s+$/g, '').replace(/\s+/g,"%20");
+    }
+
+    function appendKSToPath(tier,path,currentTier){
+        const ksTierDict = {
+            1 : "Killstreak%20",
+            2 : "Specialized%20Killstreak%20",
+            3 : "Professional%20Killstreak%20",
+        }
+        let splitPath = path.split("/");
+        splitPath[3] = splitPath[3].replace(ksTierDict[currentTier],"");
+        if(tier > 0){
+            splitPath[3] = ksTierDict[tier].concat("",splitPath[3]);
+        }
+        let tieredPath = splitPath.join("/");
+        return tieredPath;
+    }
+
+    class itemData{
```

Comment:
This is a pretty generic name, it would be nice if it would be renamed to something like `ItemStatPageInfo` (or whatever else if you have a better fitting name)